control-plane/subcommand/partition-init/command.go

```diff
@@ -62,8 +61,6 @@ func (c *Command) init() {
 	c.flags.UintVar(&c.flagServerPort, "server-port", 8500, "The HTTP or HTTPS port of the Consul server. Defaults to 8500.")
 	c.flags.StringVar(&c.flagConsulCACert, "consul-ca-cert", "",
 		"Path to the PEM-encoded CA certificate of the Consul cluster.")
```

Comment:
Removing this one too because it's not used anymore
```suggestion
```

Reply:
Done.